openpathsampling/volume.py

```diff
@@ -359,8 +366,14 @@ def __sub__(self, other):
             return super(CVDefinedVolume, self).__sub__(other)
 
     def __call__(self, snapshot):
-        l = float(self.collectivevariable(snapshot))
-        return l >= self.lambda_min and l <= self.lambda_max
+        l = self.collectivevariable(snapshot).__float__()
+        if self.lambda_min != float('nan') and self.lambda_min > l:
+            return False
+
+        if self.lambda_max != float('nan') and self.lambda_max < l:
+            return False
+
```

Comment:
> Anyway I changed it to `inf` and `-inf`

Still seeing the same things here... and the point is that it shouldn't need the nan-test here. Actually, it should be even simpler than the original, since Python understands `lambda_min <= l <= lambda_max`.